src/components/FacebookAuthenticationButton/types.ts

```diff
@@ -0,0 +1,12 @@
+import { Stream } from 'most';
```

Comment:
This looks like it have been factored out. According to @sdebaun , we aren’t allowed to do such, unless I have misunderstood the Rules of Play.